api/objects/validate

```diff
@@ -79,18 +79,31 @@ case "update-service":
     }
 
     if($data['Ports']) {
+        $portValidator = $v->createValidator(Validate::PORTNUMBER);
         if (strpos($data['Ports'][0], "-") !== false ) { # port range
-            $tmp = explode("-",$data['Ports'][0]);
-            # two non-zero integers, right number must be greater then left one
-            if ( !isset($tmp[0], $tmp[1]) || !(int)$tmp[0] || !(int)$tmp[1] || (int)$tmp[0] >= (int)$tmp[1] ) {
-                $v->addValidationError('Ports', 'not_valid_port_range');
+            # Verify that we use either a comma separated list OR a range of ports    
+            if (isset ($data['Ports'][1])) {
+                $v->addValidationError('Ports', 'must_be_port_list_or_a_port_range');
+            } else {
```

Comment:
Let's state it at the beginning: a valid port range means `count($data['Ports']) == 1 AND "-" in $data['Ports'][0]`.

...else we evaluate it as a ports list (from line [L88](https://github.com/NethServer/nethserver-firewall-base/pull/198/files#diff-08838c71de420f98db852f2960e5e3f8d11c7f94c95e9fc6f55498aeabd8afc8L88))

Reply:
done